lntest/itest/lnd_signer_test.go

```diff
@@ -411,3 +412,82 @@ func deriveCustomizedKey(ctx context.Context, node *lntest.HarnessNode,
 	}
 	return pub, nil
 }
+
+// testSignVerifyMessage makes sure that the SignMessage RPC can be used with
+// all custom ways of specifying by verifying with VerifyMessage. Tests both
+// ECDSA and Schnorr signatures.
+func testSignVerifyMessage(net *lntest.NetworkHarness, t *harnessTest) {
+	runSignVerifyMessage(t, net, net.Alice)
+}
+
+// runSignVerifyMessage makes sure that the SignMessage RPC can be used with
+// all custom ways of specifying by verifying with VerifyMessage. Tests both
+// ECDSA and Schnorr signatures.
+func runSignVerifyMessage(t *harnessTest, net *lntest.NetworkHarness,
+	alice *lntest.HarnessNode) {
+
+	ctxb := context.Background()
+	ctxt, cancel := context.WithTimeout(ctxb, defaultTimeout)
```

Comment:
Given this timeout context is used for the whole test we also expect that we can always fit all calls within?

Reply:
It should be enough with one timeout since these functions are quick. @guggero commented on this before.